src/ga_session.cpp

```diff
@@ -347,7 +347,7 @@ namespace sdk {
         , m_is_locked(false)
         , m_tx_last_notification(std::chrono::system_clock::now())
         , m_cache(m_net_params, net_params.at("name"))
-        , m_user_agent(net_params.value("user_agent", GDK_COMMIT))
+        , m_user_agent(net_params.value("user_agent", "") + " " + GDK_COMMIT)
         , m_electrum_url(
```

Comment:
If there is no 'user_agent' this will result in a leading space before the gdk commit/version - is that ok ?

Reply:
It will yes I think it's probably ok for simplicity?